win32/src/uuid_win32.c

```diff
@@ -128,3 +130,45 @@ int GUID_from_uuid(GUID* destination, const UUID_T source)
     }
     return result;
 }
+
+bool is_uuid_nil(const UUID_T uuid_value)

```

Comment:
It doesn't seem like this Windows implementation is different from the Linux version. Should we put this in a common platform neutral source file instead (to avoid forgetting to update the _other_ version in case we make some change in the future)?

Reply:
These implementations are different, the functions used to create the UUID are different.  What I did here is the same so the behavior is the same.